platform/firefox/js/platform.js

```diff
@@ -29,14 +29,11 @@ export class PlatformFirefox extends Platform {
 
     update_request_handlers(enabled, account, broker) {
         super.update_request_handlers(enabled, account, broker);
-        if (!enabled) {
-            chrome.webRequest.onBeforeSendHeaders.removeListener(
-                this.#on_before_send_headers,
-            );
-            this.account = null;
```

Comment:
Why can this line be safely dropped?

Reply:
Explanation should not be hidden in a resolved thread but rather go into a commit log.

Reply:
I added the reasoning to the commit message now as well.